pywinauto/windows/uia_element_info.py

```diff
@@ -55,6 +56,28 @@ def elements_from_uia_array(ptrs, cache_enable=False):
     return elements
 
 
+def is_element_satisfies_criteria(element, process=None, class_name=None, name=None, control_type=None,
```

Comment:
Grammar nitpicking, the name should be "is_element_satisfying_criteria" or "element_satisfies_criteria"

Reply:
done